core/snippets/snippets/commentLine.snippet

```diff
@@ -18,8 +18,3 @@ language: xml | html
 -
 <!-- $0 -->
 ---
-
-language: scm
```

Comment:
I don't think this one is necessary. scm is quite unusual and we only have a single sniper for it.